test/ktn_code_SUITE.erl

```diff
@@ -33,6 +42,14 @@ init_per_suite(Config) ->
 end_per_suite(Config) ->
     Config.
 
+enabled_features() ->
+    try
```

Comment:
Originally used `-if(?OTP_RELEASE >= 25).` here, BUT rebar3_format does not format`-else.` the same way on all tested versions.

Reply:
Yeah… because `else` is part of `maybe` now 🤷🏻